scripts/compactor_runner.py

```diff
@@ -0,0 +1,339 @@
+#!/usr/bin/env python3
+
+"""
+USAGE:
+1. To trigger compactor instantly without trimming at the end of the cycle
+/usr/share/corfu/scripts/compactor_runner.py --port <port> --compactorConfig /usr/share/corfu/conf/corfu-compactor-config.yml --instantTriggerCompaction=True
+2. To trigger compactor instantly with trimming at the end of the cycle
+ /usr/share/corfu/scripts/compactor_runner.py--port <port> --compactorConfig /usr/share/corfu/conf/corfu-compactor-config.yml --instantTriggerCompaction=True --trimAfterCheckpoint=True
+3. To freeze compactor
+/usr/share/corfu/scripts/compactor_runner.py--port <port> --compactorConfig /usr/share/corfu/conf/corfu-compactor-config.yml--freezeCompaction=True
+4. To unfreeze compactor
+/usr/share/corfu/scripts/compactor_runner.py--port <port> --compactorConfig /usr/share/corfu/conf/corfu-compactor-config.yml--unfreezeCompaction=True
+5. To Sync protobuf descriptor tables (Done when upgrading from highline to impactor/jetfire)
```

Comment:
Maybe need to remove this - "highline to impactor/jetfire"

Reply:
thank you!

Reply:
removing entire option as discussed offline, thanks